README.md

```diff
@@ -403,6 +396,40 @@ bookings: [
 ],
 ```
 
+### clickOutsideElementId
+
+- Type: `String`
+- Default: `''`
+
+Give the `id` of your element you want to click outside
```

Comment:
hmm j'ai pas bien compris. L'élément que tu veux cliquer à l'extérieur ? Tu veux dire un élément extérieur sur lequel tu cliques qui permet d'ouvrir le calendrier ?
=> Id of an element outside the DatePIcker that shouldn't close the calendar on click